### DISCUSSION TITLE
Перевод для Твича

### DISCUSSION THREAD

**Discussion by Sprinboy** [ilyhalight/voice-over-translation#954] [Q&A] [answered]
Подскажите как запустить перевод для твича? Вижу что сайт вроде поддерживается но никаких виджетов не отображается(как на ютубе), на Ютубе все отлично работает. Использую Хром

**Comment by Sprinboy** [comment]
UPD. По какой то причине все заработало и плашка вылезает с переводом однако он неможет перевести "Видео слишком длинное" или "Не найден индификатор ID видео", я так понимаю он не может переводить трансляции и записи? Или я могу как то настроить? Спасибо

**Comment by ilyhalight** [comment, marked as answer]
На твиче переводятся только клипы и видео, которые <4 часов. Перевод стримов на твиче пока не поддерживается
